http-add-on/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
```

Comment:
We should add the typical Helm labels here, but a dedicated issue to track this is OK.

Reply:
This applies to all resources

Reply:
I added some default labels based on KEDA's templates, I'll push them and you can review again

Reply:
Would be nice to have these as well:
https://github.com/kedacore/charts/blob/master/keda/templates/12-keda-deployment.yaml#L9-L14

Reply:
Added!